proxy_agent/src/redirector.rs

```diff
@@ -55,8 +56,8 @@ pub struct AuditEntry {
     pub logon_id: u64,
     pub process_id: u32,
     pub is_admin: i32,
-    pub destination_ipv4: u32,
-    pub destination_port: u16,
+    pub destination_ipv4: u32, // in network byte order
```

Comment:
We're reliant on endianness here, I think it would be safer to have 4 u8s as a struct or array

Reply:
I do not get what's the issue here, since it may not within the scope of this PR code changes, could you please help file an issue with more details? I will investigate further and may fix it later.

Reply:
to add bit more context, this AuditEntry is for the struct ebpf map, it must match the order and size defined in the ebpf program.